Makefile

```diff
@@ -49,14 +50,17 @@ ${DEPSVENV}/.done:
 	virtualenv -p python3 ${DEPSVENV}
 	source ${DEPSVENV}/bin/activate && pip install grpcio-tools==1.20.0 googleapis-common-protos==1.5.9 && deactivate
 	touch $@
+	
 deps-swift:
 	cd ${DEPSSWIFT} && wget \
-		"https://github.com/grpc/grpc-swift/archive/1.0.0-alpha.9.tar.gz" && \
-		tar xzf nio.tar.gz && \
-		mv grpc-swift-nio bin && \
-		rm -f nio.tar.gz
-
-	cd ${DEPSSWIFT}/bin && make plugins 
+		"https://github.com/grpc/grpc-swift/archive/${SWIFT_GRPC_VERSION}.tar.gz" && \
+ 		tar xzf ${SWIFT_GRPC_VERSION}.tar.gz && \
+ 		mv grpc-swift-${SWIFT_GRPC_VERSION} bin && \
+ 		rm -f ${SWIFT_GRPC_VERSION}.tar.gz
+ 		cd ${DEPSSWIFT}/bin && make plugins
+ 		cp protoc-gen-grpc-swift ${BINPATH}
+ 		cp protoc-gen-swift ${BINPATH}
```

Comment:
Where is BINPATH defined? Should this be DEPSBIN?

Reply:
Oops